### PR TITLE
Stackrox: Add version compatibility test step

### PR DESCRIPTION
**Pull request by fredrb**
New test step introduced in stackrox/stackrox: https://github.com/stackrox/stackrox/pull/2771

### CODE REVIEW

**Comment by gavin-stackrox** [comment]
/ok-to-test
@fredrb you should probably add yourself to the openshift org (so that you can run CI yourself) https://docs.engineering.redhat.com/display/StackRox/Onboarding+to+OpenShift+GitHub+Organization

**Comment by fredrb** [comment]
@gavin-stackrox: could you check again?

File: ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml [hunk 1]
```diff
@@ -506,6 +506,57 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )gke-upgrade-tests,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (gavin-stackrox) [inline ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml:510]:
```suggestion
    always_run: false
```
This has to be manually set after `make update` or `make jobs` if you do not want the job to run on all PRs. I would expect that at least initially you only want this on demand i.e. via `/test gke-version-compatibility-tests`.

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml (hunk 1 above)
Comment (fredrb) [reply ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml:510]: > I would expect that at least initially you only want this on demand i.e. via /test gke-version-compatibility-tests.

Exactly, thanks!

**Pull request merged**